### ISSUE TITLE
Are new versions automatically deployed?

### ISSUE THREAD

**Issue by ddeboer** [proycon/codemeta-harvester#15]
The Network of Terms is still not being harvested correctly:

I added:

- https://github.com/CLARIAH/tool-discovery/blob/master/source-registry/network-of-terms-graphql.yml
- https://github.com/CLARIAH/tool-discovery/blob/master/source-registry/network-of-terms-reconciliation.yml

But all I’m seeing is https://tools.clariah.nl/network-of-terms/1.0.1, an outdated version. 

This may be caused by my latest commit 7b0ec2fdd6a2d0ef28ba32acee14ec006c93bde3 not being live yet. Is that the case?

**Comment by proycon** [comment]
> This may be caused by my latest commit https://github.com/proycon/codemeta-harvester/commit/7b0ec2fdd6a2d0ef28ba32acee14ec006c93bde3 not being live yet. Is that the case?

Yes, sorry, it was merged but not released yet. I'll try to get it done tomorrow so everything's nice and ready for the tech day.

**Comment by proycon** [comment]
The development version is deployed now, does it look okay to you? https://tools.dev.clariah.nl

I'll then release and deploy the production release as well.

**Comment by ddeboer** [comment]
Looks better on https://tools.dev.clariah.nl/. Two points:

1. The old https://tools.dev.clariah.nl/network-of-terms/1.0.1 is still there. How do we remove sources?
2. Not all info (or none?) from the codemeta.json seems to be picked up, for example name and readme link of  https://tools.dev.clariah.nl/network-of-terms-graphql/@netwerk-digitaal-erfgoed/network-of-terms-graphql@1.55.0 vs https://github.com/netwerk-digitaal-erfgoed/network-of-terms/blob/master/packages/network-of-terms-graphql/codemeta.json
